### ISSUE TITLE
Rephrase error message for people logging in with an existing user email

### ISSUE THREAD

**Issue by sstead** [openfoodfoundation/openfoodnetwork#1611]
See suggested rephrase below:
![image](https://user-images.githubusercontent.com/10843056/26956353-4b4a00c0-4d00-11e7-98fe-d99bac4508e0.png)

**Comment by diogomatsubara** [comment]
Hi, I'm new to RoR but I'd like to give this one a try.
I grepped the code base and couldn't find the 'has already been taken' error, so I suspect it might be coming from one of the dependencies. It doesn't show up in the locales files either and when I use a different locale (pt-BR) it shows up correctly translated. Could someone more knowledgeable about the code base give some pointers about this one?

**Comment by enricostano** [comment]
Hi @diogomatsubara and welcome to the OFN community!

Your suspects are correct 😂 , that copy comes from a dependency we use to manage user authentication: the `devise` gem.

1. we require [spree_auth_devise](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/Gemfile#L45) first, a wrapper of the final gem
2. that wrapper defines our `User` model and uses `devise` [capabilities](https://github.com/spree/spree_auth_devise/blob/1-3-stable/app/models/spree/user.rb#L5). The dependency is declared [here](https://github.com/spree/spree_auth_devise/blob/1-3-stable/spree_auth_devise.gemspec#L20)
3. finally you can explore how Devise work in the gem [documentation](https://github.com/plataformatec/devise/tree/v2.2.3)

Basically the `devise` gem adds unique email validation to the `User` model. Give a look to `ActiveRecord` validations [documentation](http://guides.rubyonrails.org/active_record_validations.html#working-with-validation-errors) to have a better understanding.

As you can see a copy in the UI boils down to a model validation that will pick a standard Rails copy like `"#{field} has already been taken"`. Defined in [rails-i18n gem](https://github.com/svenfuchs/rails-i18n/blob/rails-2-3/rails/locale/en-US.yml#L208).

How to override it? Something like [this](https://stackoverflow.com/questions/2652173/rails-override-validator-message?answertab=votes#tab-top) should work.

Please ping us back if you need more information!

**Comment by daniellemoorhead** [comment]
@diogomatsubara welcome 😄 

Just to let you know, we have a slack team and a dev channel where people working on the codebase chat with each other. It's a great place to get some help as you start working on stuff. Let me know if you'd like us to send through an invitation to join. We also have community message boards where things get discussed [here](https://community.openfoodnetwork.org/).

Cheers!

**Comment by diogomatsubara** [comment]
@enricostano Thanks for the pointers! They were really helpful. I submitted a PR with a fix. Let me know what you think
@daniellemoorhead Thanks, I joined the dev channel.
